src/sec_edgar_bulker/core/header_generator.py

```
@@ -0,0 +1,95 @@
+"""Header generator for SEC EDGAR requests."""
+
+import random
+import re
+from typing import Dict
```

Ruff (0.8.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)